src/app/shared/components/template/components/radio-button-grid/radio-button-grid.component.ts

```diff
@@ -27,6 +27,8 @@ interface IRadioButtonGridParams {
   grid_gap: string;
   /* TEMPLATE PARAMETER: "variant". The style variant of the button_grid */
   variant?: "default" | "circle-icon";
+  /* TEMPLATE PARAMETER: "style". Defines the display style of the buttons. Default is grid */
+  style?: "grid" | "flex";
```

Comment:
nit(blocking): I think these should be variants really as opposed to styles. In that case, the easiest way to handle `grid` as the default is probably just to treat it as default in the styling, i.e. scope the current grid styling to apply by default, and override it with the flex styling in the case that `[data-variant~="flex"]`.

@chrismclarke do you agree that this should be a variant? In some sense these properties are orthogonal (in theory we could support `circle-icon` and `flex` in combination, for example, but that should be achievable with variants by just setting both, e.g. `variant: circle-icon flex`